src/librustdoc/clean/types.rs

```diff
@@ -599,16 +599,16 @@ impl Item {
         })
     }
 
-    crate fn stable_since(&self, tcx: TyCtxt<'_>) -> Option<SymbolStr> {
+    crate fn stable_since(&self, tcx: TyCtxt<'_>) -> Option<Symbol> {
```

Comment:
Perhaps at this point it's too late, but it'd be better to separate this refactor into a separate commit.

Reply:
I can make it into a separate commit.